registration/registration_tests.py

```diff
@@ -211,10 +214,68 @@ def test_registration__21(db, user):
     assert not testuser.has_perm('registration.verified')
 
 
+def test_registration__22(browser, user, mail_outbox):
+    """User can download a plain application form."""
+    register_with_pwd(browser, user, mail_outbox)
+    _log_in(browser, user['email'], PWD)
+    browser.open(APPLY_URL)
+    browser.getControl('Apply').click()
+    # TODO don't checks content of pdf
+    assert browser.headers['Content-Type'] == 'application/pdf'
+
+
+def test_registration__23(browser, user, mail_outbox):
+    """User can download an automatically created application form."""
+    register_with_pwd(browser, user, mail_outbox)
+    _log_in(browser, user['email'], PWD)
+    browser.open(APPLY_URL)
+    browser.getControl('application form').click()
+    # TODO don't checks content of pdf
```

Comment:
You could check for a quick `assert browser.contents.startswith('%PDF-1.4')` that at least a PDF is returned.

Reply:
I actually added tests for the PDF content in https://github.com/gocept/ok_tools/pull/19/commits/352618f610a491af4528ec8f3d9ae3f540a97f69. I thought the fact that a PDF is returned would be already checked by the content-typ.

Reply:
The Content-Type in the HTTP header is just, what was written into the header. This is not necessarily, what the browser actually sent.